aws/table_aws_sns_topic.go

```diff
@@ -336,15 +344,23 @@ func listTagsForSnsTopic(ctx context.Context, d *plugin.QueryData, h *plugin.Hyd
 
 func snsTopicTurbotTags(_ context.Context, d *transform.TransformData) (interface{}, error) {
 	tags := d.HydrateItem.(*sns.ListTagsForResourceOutput)
-	// if !ok {
-	// 	return nil, nil
-	// }
-	var turbotTagsMap map[string]string
-	if tags.Tags != nil {
-		turbotTagsMap = map[string]string{}
-		for _, i := range tags.Tags {
-			turbotTagsMap[*i.Key] = *i.Value
-		}
+	if len(tags.Tags) == 0 {
+		return nil, nil
+	}
+
+	turbotTagsMap := map[string]string{}
+	for _, i := range tags.Tags {
+		turbotTagsMap[*i.Key] = *i.Value
 	}
+
 	return turbotTagsMap, nil
 }
+
+func formatSnsTopicEmptyTags(_ context.Context, d *transform.TransformData) (interface{}, error) {
```

Comment:
What formatting are we doing in this function? Here the function name starts with `format`, while in `aws_iam_user` table, the transform functions start with `handle`. Should this one also use the word `handle` to be consistent, or is there some formatting occurring?

Reply:
As per discussion, we'll change this function name to be `handleSnsTopicEmptyTags` to be consistent